parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -186,6 +186,7 @@ installAzureCNI() {
         downloadAzureCNI
     fi
     tar -xzf "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" -C $CNI_BIN_DIR
+    chown -R root:root $CNI_BIN_DIR
```

Comment:
Thanks for the quick suggestion Ace
The "Which issue(s) this PR fix" seems relevant.
Shoudd everything isntalled by CSE be root or do we really believe just cni is special?  Surely we're doing other tar -xzf's here....

Reply:
> Surely we're doing other tar -xzf's here

we are

> Shoudd everything isntalled by CSE be root
I think generally, yes (I think if anything needs to be accessible by non-root can use file permissions?)

> do we really believe just cni is special?

well, it's the only one which broke users ;) 

also, given this was due to a change in build infra and this is one of the few deps upstream doesn't build, I think it _is_ special in some sense (acknowledging the weakness of that argument in practice since upstream could have the same issues).

Reply:
oh, I noticed we do the chown in setupCNIDirs, it just happens before we untar.